### PR TITLE
Internal: Implement new 'Latest' channel [ED-11967]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: .github/workflows/one-click-hosting-release.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Hi All :smile:\n\nWe just Published Could release of *Elementor*\n`${{ env.PACKAGE_VERSION }}`"
+                    "text": "Hi All :smile:\n\nWe just Published a Cloud release of *Elementor*\n`${{ env.PACKAGE_VERSION }}`"
```
Comment (TzviRabinovitch) [inline .github/workflows/one-click-hosting-release.yml:99]: why not change to Hosting?

**Review by TzviRabinovitch — DISMISSED** [review]

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/latest-release.yml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      - main
+
+# Only cancel in-progress jobs or runs for the current workflow run
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
```
[the remark this answers is not in the document]
Comment (davseve) [reply .github/workflows/latest-release.yml]: Shouldn't the `github.base_ref` be main`?

File: .github/workflows/latest-release.yml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+# Only cancel in-progress jobs or runs for the current workflow run
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  latest_release:
+    if: github.event.pull_request.merged == true && github.base_ref == 'main' && !contains(github.event.pull_request.labels.*.name, 'skip release')
+    runs-on: ubuntu-22.04
+    steps:
```
Comment (Barcov) [inline .github/workflows/latest-release.yml:22]: I'd suggest checking if the PR was merged. The action can fire for closed and unmerged PRs as well.

```suggestion
    if: github.event.pull_request.merged == true
    steps:
```

File: .github/workflows/bump-channel-version/action.yml [hunk 4]
```diff
@@ -21,8 +21,14 @@ runs:
   steps:
     - shell: bash
       run: |
-            CURRENT_CHANNEL_VERSION=${{ env.CURRENT_CHANNEL_VERSION }}
-            NEW_CHANNEL_VERSION=$((CURRENT_CHANNEL_VERSION+1))
+
+            if [[ ${{ inputs.CHANNEL }}  == "latest" ]]; then
```
Comment (Barcov) [inline .github/workflows/bump-channel-version/action.yml:25]: I'd suggest we restructure this section to make it easier to maintain. 

```suggestion
function GetNextChannelNumber() {
  case $1 in

  * )
    CURRENT_CHANNEL_VERSION=${{ env.CURRENT_CHANNEL_VERSION }}
    NEW_CHANNEL_VERSION=$((CURRENT_CHANNEL_VERSION+1))
    ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}-${{ inputs.CHANNEL }}${NEW_CHANNEL_VERSION}
    ;;&

  "latest" )
     NEW_CHANNEL_VERSION=-$(date +%s)
     ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}-${{ inputs.CHANNEL }}${NEW_CHANNEL_VERSION}
    ;;

  "ga" )
    ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}
    ;;
  esac

}

GetNextChannelNumber ${{ inputs.CHANNEL }}
```

File: .github/workflows/latest-release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (yotamselementor) [reply .github/workflows/latest-release.yml]: Yes, sorry - copied from another GH Action

File: .github/workflows/latest-release.yml [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  cancel-in-progress: true
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-22.04
+    steps:
+      - run: |
+          echo ${{ github.event.pull_request.merged }} was succesfully merged
+  latest_release:
```
[the remark this answers is not in the document]
Comment (davseve) [reply .github/workflows/latest-release.yml:19]: I've removed the redundant step and added a new condition

File: .github/workflows/bump-channel-version/action.yml (hunk 4 above)
Comment (davseve) [reply .github/workflows/bump-channel-version/action.yml]: Implemented

File: .github/workflows/latest-release.yml (hunk 3 above)
Comment (davseve) [reply .github/workflows/latest-release.yml:22]: Done

**Review by yotamselementor — DISMISSED** [review]

**Review by Barcov — APPROVED** [review]

**Review by Barcov — DISMISSED** [review]

**Review by Barcov — APPROVED** [review]

**Pull request merged**